init-letsencrypt.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+read -p "Enter your domain name (example.com) : " DOMAIN_NAME
+domains=($DOMAIN_NAME evalai.$DOMAIN_NAME evalapi.$DOMAIN_NAME)
```

Comment:
this can be change to only `domains=$DOMAIN_NAME`. Accept both the domain names not just the suffix. We can't add fixed prefix

Reply:
We're iterating over all domains & including it in domain args. This is specified to certbot when requesting for certs in final call. So, `domain=$DOMAIN_NAME` would be incorrect I believe

Reply:
But you can accept array of domain names as a input and assign it to `DOMAIN_NAME`

Reply:
then how would we differentiate in frontend & backend domains ?
I added a read command for explicitly taking frontend & backend domains as input so now you won't have to fiddle around with the nginx config. You can check that